pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Consumer.java

```diff
@@ -233,7 +236,7 @@ public boolean readCompacted() {
             writePromise.setSuccess(null);
             return writePromise;
         }
-
+        int unackedMessage = totalMessages;
```

Comment:
Typo: change to unackedMessages (plural)

Reply:
Thanks